src/applications/coronavirus-research/sign-up/containers/ConsentFormContent.jsx

```diff
@@ -6,49 +6,106 @@ export function ConsentLabel() {
+      <strong>Will I get paid? </strong>
+      <p>You will not be paid to participate in this Volunteer List </p>
+      <strong>Will I have to pay anything?</strong>
+      <p>You will not have to pay anything to be in this study. </p>
+      <strong>Do I have to be in this study?</strong>
+      <p>
+        It’s always your choice to answer any questions we may ask or not
+        indicate consent by exiting this page. You do not have to participate in
+        this study.{' '}
+      </p>
+      <strong>Can I change my mind later and stop being in this study? </strong>
+      <p>
+        You can withdraw your name from this list at any time by emailing
+        research@va.gov. Your decision to join or not join this list won’t
+        affect your VA health care or any of your VA benefits or services in any
+        way. If you submit your responses, we will keep your information as part
+        of the Volunteer List indefinitely, or until the project ends or you
+        request to be withdrawn.
+      </p>
+      <strong>Will my information be protected from the public? </strong>
+      <p>
+        We’ll include your information in our secure electronic database and
+        will keep your name and all the information you tell us in this study as
+        confidential as possible. We’ll only give database access to people with
+        permission. Your information will be made available to researchers who
+        need your information for VA COVID-19 studies. We will always check with
+        you before sharing your information with researchers looking for
+        volunteers.
+      </p>
+      <p>
+        Also, other federal agencies as required, such as the VA Office of
+        Research Oversight and the VA Office of the Inspector General may have
+        access to your information.{' '}
+      </p>
+      <strong>
+        Who can I talk to if I have questions about the research, problems
+        related to the study or if I think I’ve been hurt by being a part of the
+        study?
+      </strong>
+      <p>
+        If you have any questions, concerns or complaints about this research
+        study, its procedures, risks and benefits, or alternative courses of
+        treatment, you can contact the Volunteer List at research@va.gov. You
+        should also contact the Volunteer List at any time if you feel you have
+        been hurt by being a part of this study.
+      </p>
+      <p>
+        If you are not satisfied with how this study is being conducted, or if
+        you have any concerns, complaints, or general questions about the
+        research or your rights as a participant, and would like to speak
+        someone independent of the research team please contact the Stanford
+        Institutional Review Board (IRB) at{' '}
+        <va-telephone contact="6507235244" /> or toll free at{' '}
+        <a href="tel:1-866-680-2906" aria-label="1 8 6 6 6 8 0 2 9 0 6">
+          1-866-680-2906
+        </a>
```

Comment:
```suggestion
        <va-telephone international contact="8666802906" />
```

Reply:
@bkjohnson I'll take a look before merging, but I'm my local testing I had since issue with it being an 1-866 number. I'll double your suggestions when I back at my desk.

Reply:
Awesome @bkjohnson , that works! Thanks again!